buildSrc/src/main/groovy/org/elasticsearch/gradle/test/DistroTestPlugin.java

```diff
@@ -65,6 +70,40 @@
 import static org.elasticsearch.gradle.vagrant.VagrantMachine.convertWindowsPath;
 
 public class DistroTestPlugin implements Plugin<Project> {
+    private static final Logger logger = Logging.getLogger(DistroTestPlugin.class);
+    /**
+     * This plugin generates test tasks for the various VMs in :qa:os. The
+     * Docker packaging tests are run on all of them, apart from those
+     * listed here. These values correspond to the sub-project names, which
+     * correspond to the directory names.
+     */
+    private static final List<String> DOCKER_VM_EXCLUDE_LIST = List.of(
+        "centos-6",
+        "debian-8",
+        "oel-6",
+        "oel-7",
+        "opensuse-42",
+        "sles-12",
+        "windows-2012r2",
+        "windows-2016"
+    );
+
+    /**
+     * This plugin generates a number of test tasks, some of which are
+     * Docker packaging tests. When running on the host OS i.e. not in a
+     * VM, only certain operating systems are supported. This list
+     * specifies the Linux OS variants that are tested. The actual question
+     * of whether to run the Docker tests is answered by {@link
+     * #shouldRunDockerTests()}.
+     */
+    private static final List<String> DOCKER_LINUX_INCLUDE_LIST = List.of(
```

Comment:
My comment above applies here as well.

Also, isn't this essentially the inverse of the above list?